lesson09/main.js

```diff
@@ -31,25 +31,21 @@ function renderMenus(menus){
   ul.appendChild(fragment);
 }
 
-function getMenus() {
+const getMenus = new Promise((resolve)=> {
   renderCircle();
   const menus = [ 
     {to: "bookmark.html", img:"img/1.png", alt:"画像1", text: "ブックマーク"}, 
     {to: "message.html", img:"img/2.png", alt:"画像2", text: "メッセージ"} 
   ]; 
   setTimeout(() => {
-		console.log(menus);
-    return menus;
+    resolve(menus);
   }, 3000);
-}
+})
 
-async function men() {
-	 await getMenus();
-	 removeCircle();
+async function deployMenus() {
+	const menus = await getMenus;
+	await removeCircle();
+	await renderMenus(menus);
 }

```

Comment:
MUST
The `removeCircle()` and `renderMenus(menus)` functions do not have async, so you cannot await them. 
I also do not think it is necessary to AWAIT.
```suggestion
async function deployMenus() {
	const menus = await getMenus;
	removeCircle();
	renderMenus(menus);
}

```

Reply:
Thank you mai-san :)
I didn't understand about await ;<
I learned about await :)